cmd/ocsp-responder/main_test.go

```diff
@@ -285,3 +286,9 @@ func TestExpiredUnauthorized(t *testing.T) {
 	_, _, err = src.Response(ocspReq)
 	test.AssertEquals(t, err, bocsp.ErrNotFound)
 }
+
+func TestKeyHashing(t *testing.T) {
+	src, err := makeDBSource(expiredSelector{}, "./testdata/test-ca.der.pem", []string{"00"}, time.Second, blog.NewMock())
```

Comment:
Can you use `mockSelector` here? It would be clearer since we don't depend on the `expiredSelector` behavior specifically.